modules/javafx.graphics/src/main/native-prism/SwingInterop.c

```diff
@@ -24,14 +24,15 @@
  */
 
 #include <jni.h>
+#include "com_sun_glass_ui_Application.h"
```

Comment:
unrelated: do we have steps to ensure that the JNI headers are always included, or it is up to the reviewers?

Reply:
It's up to the reviewers. And short of writing a custom scanner (which I don't think is worth the effort) there really isn't a good way to automate the check.